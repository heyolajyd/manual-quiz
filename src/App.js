import React, { useState } from 'react';
import { ThemeProvider } from 'styled-components';
import Quiz from 'pages/Quiz';
import { theme } from './styles/global';
import Landing from 'pages/Landing';

const App = () => {
  const [isQuizStarted, setIsQuizStarted] = useState(false);

  return (
    <ThemeProvider theme={theme}>
      <>
        {isQuizStarted ? (
          <Quiz />
        ) : (
          <Landing startQuiz={() => setIsQuizStarted(true)} />
        )}
      </>
    </ThemeProvider>
  );
};

export default App;
