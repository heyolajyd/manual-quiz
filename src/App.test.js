import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import 'jest-styled-components';
import App from './App';

describe('App', function () {
  it('should match snapshot', () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
