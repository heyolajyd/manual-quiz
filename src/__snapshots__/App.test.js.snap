// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div>
  <main>
    <section
      class="sc-fznxsB ljSLYw"
    >
      <svg
        class="sc-fznKkj iiymtR"
      >
        manual.svg
      </svg>
      <h1
        class="sc-fznJRM kemBEs"
      >
        <div>
          Be good
        </div>
        <div>
          to yourself
        </div>
      </h1>
      <article>
        <p>
          We’re working around the clock to bring you a holistic approach to your wellness. From top to bottom, inside and out.
        </p>
      </article>
      <button
        class="sc-AxiKw kupFjX"
      >
        TAKE THE QUIZ
      </button>
    </section>
    <section
      class="sc-fznWqX hTbzEW"
    >
      <h2>
        What we can help with
      </h2>
      <div
        class="sc-fzoiQi cGTzoi"
      >
        <img
          alt="hair-loss"
          src="hair-photo.png"
        />
        <div
          class="sc-fzqARJ drQgeY"
        >
          <span
            class="sc-fzqNqU iWOUzw"
          >
            01
          </span>
          <article
            class="sc-fzoyTs bxOeBk"
          >
            <h6
              class="sc-fzoNJl zGTud"
            >
              Hair Loss
            </h6>
            <h5>
              Hair loss needn’t be irreversible. We can help!
            </h5>
            <p>
              Hair loss needn’t be irreversible. There’s a scientifically proven way that’s most effective in keeping and regrowing your hair. It’s all to do with blocking a pesky hormone called DHT. That’s the bad guy behind hair loss. And you can keep him at bay. The choice is yours.
            </p>
          </article>
        </div>
      </div>
      <div
        class="sc-fzoiQi cGTzoi reversed"
      >
        <div
          class="sc-fzqARJ drQgeY"
        >
          <span
            class="sc-fzqNqU iWOUzw"
          >
            02
          </span>
          <article
            class="sc-fzoyTs bxOeBk reversed"
          >
            <h6
              class="sc-fzoNJl zGTud"
            >
              Erectile Dysfunction
            </h6>
            <h5>
              Erections can be a tricky thing. But no need to feel down!
            </h5>
            <p>
              There are plenty of reasons why you might be having difficulty in the erection department. We can help you figure out possible reasons why. And prescribe a pill if needed.
            </p>
          </article>
        </div>
        <img
          alt="sex"
          src="sex-photo.png"
        />
      </div>
    </section>
    <footer
      class="sc-fznZeY jHGQzG"
    >
      <div
        class="row"
      >
        <div
          class="sc-fzokOt ijGnNm col-sm-3 col-12"
        >
          <svg
            class="sc-fznKkj fAZOGT"
            height="75"
            width="80"
          >
            manual.svg
          </svg>
        </div>
        <div
          class="col-sm-2 col-4"
        >
          <h5
            class="sc-fzqBZW iwzNxE"
          >
            Product
          </h5>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Popular
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Trending
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Guided
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Products
          </a>
        </div>
        <div
          class="col-sm-2 col-4"
        >
          <h5
            class="sc-fzqBZW iwzNxE"
          >
            Company
          </h5>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Press Releases
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Mission
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Strategy
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            About
          </a>
        </div>
        <div
          class="col-sm-2 col-4"
        >
          <h5
            class="sc-fzqBZW iwzNxE"
          >
            Info
          </h5>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Support
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Customer Service
          </a>
          <a
            class="sc-fzqNJr kvyoFZ"
          >
            Get Started Guide
          </a>
        </div>
        <div
          class="col-sm-3 col-12"
        >
          <h5
            class="sc-fzqBZW iwzNxE"
          >
            Follow us
          </h5>
          <div
            class="sc-fzoyAV jyjPxR"
          >
            <span
              class="sc-fzoLag iwpCUW"
            >
              <svg>
                facebook.svg
              </svg>
            </span>
            <span
              class="sc-fzoLag iwpCUW"
            >
              <svg>
                twitter.svg
              </svg>
            </span>
            <span
              class="sc-fzoLag iwpCUW"
            >
              <svg>
                linkedin.svg
              </svg>
            </span>
          </div>
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="col-sm-12"
        >
          <hr
            class="sc-fzoXzr buwQm"
          />
        </div>
        <div
          class="col-sm-12"
        >
          <div
            class="sc-fzpjYC cDFfOb"
          >
            © 2019 Manual. All rights reserved.
          </div>
        </div>
      </div>
    </footer>
  </main>
</div>
`;
